config/300-taskrun.yaml

```diff
@@ -26,20 +26,13 @@ spec:
     - tekton-pipelines
   scope: Namespaced
   additionalPrinterColumns:
```

Comment:
```go
  additionalPrinterColumns:
  - name: Succeeded
    type: string
    JSONPath: ".status.conditions[?(@.type==\"Succeeded\")].status"
  - name: Reason
    type: string
    JSONPath: ".status.conditions[?(@.type==\"Succeeded\")].reason"
  - name: StartTime
    type: date
    JSONPath: .status.startTime
  - name: CompletionTime
    type: date
    JSONPath: .status.completionTime
```

Reply:
I guess when it's not succeeding it's going to be blank, i wonder if there is a better way to do that?

Reply:
@chmouel I would hope there is a better way. That said, I think for the "automatic and free"  fromatting that k8s/kubectl give us, it's already ok :sweat_smile:. And we also want to experiment in having a proper cli that would handle those properly :angel: